torchbenchmark/models/squeezenet1_1/__init__.py

```diff
@@ -40,32 +38,22 @@ def __init__(self, device=None, jit=False, train_bs=64, eval_bs=16):
     def get_module(self):
         return self.model, self.example_inputs
 
-    # vision models have another model
-    # instance for inference that has
-    # already been optimized for inference
-    def set_eval(self):
-        pass
-
-    def train(self, niter=3):
+    # Temporarily disable training because this will cause CUDA OOM in CI
+    # TODO: re-enable this test when better hardware is available
+    def train(self, niter=1):
+        raise NotImplementedError("Temporarily disable training test because it causes CUDA OOM on T4")
         optimizer = optim.Adam(self.model.parameters())
         loss = torch.nn.CrossEntropyLoss()
         for _ in range(niter):
-            optimizer.zero_grad()
-            pred = self.model(*self.example_inputs)
-            y = torch.empty(pred.shape[0], dtype=torch.long, device=self.device).random_(pred.shape[1])
-            loss(pred, y).backward()
-            optimizer.step()
+            for _ in range(self.epoch_size):
+                optimizer.zero_grad()
+                pred = self.model(*self.example_inputs)
+                y = torch.empty(pred.shape[0], dtype=torch.long, device=self.device).random_(pred.shape[1])
+                loss(pred, y).backward()
+                optimizer.step()
```

Comment:
if you're doing hierarchical batching, `zero_grad` and `.step()` should be outside of the loop. Otherwise this code is just doing the same thing as before :)

Reply:
Thanks for the comment! Sorry I misunderstood how to do the hierarchical batching, I will create another PR to fix it.